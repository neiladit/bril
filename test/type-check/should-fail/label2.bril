#Labels are repeated
main{

    v0: int = const 1;
    v1: int = const 3;
    v3: int = add v1 v2;
    jmp doubletrap;
    v4: int = const 4;
    print v4;
    doubletrap:
    check: bool =  eq v3 v4;
    print check;
    doubletrap:
    v5: int = const 5;
    print v5;
}
