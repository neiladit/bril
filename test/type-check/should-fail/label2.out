line 12: label doubletrap show up twice
label error!
